### ISSUE TITLE
General DI Guidelines

### ISSUE THREAD

**Issue by ryananderson0219** [theqvd/theqvd#44]
I'm am searching for some basic guidelines for setting up a DI...other than the document provided. (It doesn't seem to have the information I need. I seem to have the installation of the servers good, but it's my DIs that I'm having trouble with.

For instance...I am trying to create a Lubuntu 16.10 DI. However; I'm getting:
perl-qvd-vma : Depends: qvd-nxagent but it is not going to be installed.

I also seem to be having trouble with different Desktop Environments. I would like to be able to create a Lubuntu Image, Ubuntu Gnome, and Ubuntu Mate.

Basically, I would like to know what OS versions are compatible and which desktops are compatible and how to set these up. (IE...what do I need for x.session).

Thank you,
Ryan

**Comment by ryananderson0219** [comment]
Has anyone had created a successful DI with anything newer than Ubuntu 12.04? If so, what image are you starting with?

**Comment by jzeaherra** [comment]
Hi Ryan,

  What repository did you configure in apt?

Regards,
Juan

**Comment by ryananderson0219** [comment]
Sorry for the delayed response. I am using this repository:
http://theqvd.com/packages/ubuntu QVD-3.5.0 main

Are there others?

**Comment by jzeaherra** [comment]
Hi, 

   I have just double-checked the repository and it seems fine. Could you please try to install qvd-nxagent alone to see the error you're getting?

Maybe there's a conflict with something you have previously installed.

Regards,
Juan

**Comment by ryananderson0219** [comment]
Closing issue as it seems that the issues are inconsistent with different versions of Ubuntu, and also it resolved itself once I reimaged the computer.
